runtime/lua/vim/treesitter.lua

```diff
@@ -286,6 +286,52 @@ function M.get_captures_at_cursor(winnr)
   return captures
 end
 
+--- Returns the smallest named node at the given position
+---
+---@param opts table|nil Optional keyword arguments:
+---             - bufnr integer|nil Buffer number (nil or 0 for current buffer)
+---             - pos table|nil 0-indexed (row, col) tuple. Defaults to cursor position in the
+---                             current window. May not be omitted if {bufnr} is not the current
+---                             buffer
+---             - lang string|nil Parser language
```

Comment:
Can we remove the lang option? This shouldn't be needed with the PR I merged earlier.

Reply:
Removed in e0a751d87d2c9fa84518c44343834a489dac6533.